app/views/admin/administrators/administrators/delete.html.erb

```diff
@@ -1,6 +1,8 @@
-<h1>Do you want to delete this user?</h1>
-<p class="govuk-!-font-weight-bold">Admin user: <span class="govuk-!-font-weight-regular"><%= @administrator.full_name %></span></p>
+<h1>Do you want to delete this admin user?</h1>
```

Comment:
Double check with Shani if the messaging is right. As long as she is happy it should be ok